[MM-55746] Add documentation on how to use new `ServeMetrics` plugin hook (#1325)
mattermost/mattermost-developer-documentation: site/content/integrate/plugins/best-practices.md

```

Old servers won't do anything with new, unrecognized fields, but also won't break if they are present.

## How to expose performance metrics for a plugin?

> Starting in Mattermost v9.4 we provide a new [`ServeMetrics`]({{< ref "/integrate/reference/server/server-reference#API.ServeMetrics" >}}) hook that can be used to expose performance metrics in the [open metrics format](https://openmetrics.io/) under the common HTTP listener controlled by the [`MetricsSettings.ListenAddress`](https://docs.mattermost.com/configure/environment-configuration-settings.html#listen-address-for-performance) config setting.

Reviewer: ```suggestion
From Mattermost v9.4, a [`ServeMetrics`]({{< ref "/integrate/reference/server/server-reference#API.ServeMetrics" >}}) hook can be used to expose performance metrics in the [open metrics format](https://openmetrics.io/) under the common HTTP listener controlled by the [`MetricsSettings.ListenAddress`](https://docs.mattermost.com/configure/environment-configuration-settings.html#listen-address-for-performance) config setting.
```
